src/pug/includes/body.pug

```diff
@@ -31,34 +30,37 @@ include ./fork.pug
 							option(value="DIFFICULTY_DESCENDING" class="difficulty") Difficulty, reversed
 					span order
 
-			.bottom
 				.item.flex
 					select#tags
 
-		.content.flex#content
-			- let difficulty;
-			each swag in swagList
-				- difficulty = swag.difficulty + ' difficulty';
-				- difficultyTitle = 'Difficulty: ' + swag.difficulty;
-				div(
-					class="item visible " + swag.difficulty + swag.tags.map(t => ` tag-${t}`).join('')
-					data-name=swag.name.toLowerCase()
-					data-difficulty=['easy','medium', 'hard'].indexOf(swag.difficulty)
-				)
-					div.title.flex
-						h1=swag.name
-						div(class=difficulty title=difficultyTitle)
-							span.sr-only=difficultyTitle
-					p.swag
-						each swagTag in swag.tags
-							span!=swagTag
-					div.flex.img-container
-						picture
-							source(type="image/webp" srcset=swag.images.webp)
-							source(type="image/jpeg" srcset=swag.images.jpeg)
-							img(src=swag.images.jpeg alt=swag.name + " swag you can get")
-					p.description!=swag.description
-					a(href=swag.reference) Check it out
+				.item.flex
+					input(type="checkbox" id="expired")
+					label(for="expired") Show expired opportunities
+
+			.content.flex#content
```

Comment:
It looks like the indentation was changed here - was that intentional?

Reply:
Nope

Reply:
I think I fixed this in 5b866aa342ff455c36bd3424aa5bb441f686fa49.